packages/app-project/src/components/Head/HeadContainer.js

```diff
@@ -52,6 +57,7 @@ class HeadContainer extends Component {
       <Head
         description={this.getProjectDescription()}
         ogImage={this.getProjectImage()}
+        pageTitle={this.getPageTitle()}
```

Comment:
Could this be `pageTitle={this.props.pageTitle}`? I'm not sure why it's calling a method that just returns a prop.